docs/concepts/07-chains.mdx

```diff
@@ -5,59 +5,19 @@ sidebar_position: 7
 ---
 
 import Link from "@docusaurus/Link";
+import Chains from "@site/src/components/organisms/Chains";
 import { Links } from "@site/src/constants";
 
-Sablier Protocol is live on 25+ mainnets and 11+ testnets. However, not all of these chains may be supported by the
+Sablier Protocol is live on 25+ mainnets and 11+ testnets, although not all of these chains are supported by the
```

Comment:
We can also programatically update the chain count.

Reply:
Hah. Noice!

Reply:
Or, even better, just do this:

```
import Link from "@docusaurus/Link";
import { mainnets, testnets } from "@sablier/deployments";
import Chains from "@site/src/components/organisms/Chains";
import { Links } from "@site/src/constants";

Sablier Protocol is live on {mainnets.length} mainnets and {testnets.length} testnets, although not all of these chains are supported by the
[Sablier Interface](https://app.sablier.com/). If you're interested in having your chain listed in the Interface, fill out <Link href={Links.Forms.CALL}>this form</Link>.
```